### PR TITLE
routes: take fbid for commitment

### PR DESCRIPTION
**Pull request by compnerd**

### CODE REVIEW

File: lib/routes/commitments.js [hunk 1]
```diff
@@ -2,10 +2,21 @@ const router = require('koa-router')()
...
+    let fbid = ctx.params.fbid
+    await models.db.participant.findOne({
+      where: {fbid}
+    }).then(async (participant) => {
+      if (participant) {
+        ctx.body = await models.db.participant_event.findAll({
+          where: {participant_id: participant.id}
+        })
+      } else {
+        ctx.status = 204
```
Comment (alykhank) [inline lib/routes/commitments.js]: I think this should actually return 400 to match the other "does not exist" errors

Comment (compnerd) [reply lib/routes/commitments.js]: That sounds reasonable.  Changed.

File: test/commitments-specs.js [hunk 2]
```diff
@@ -8,17 +8,23 @@ beforeEach(function syncDB () {
 })
 
 describe('commitments', () => {
-  context('GET /commitments/participant/:participant', () => {
-    it('should return empty if no commitments for participant with id=participant', async () => {
+  context('GET /commitments/participant/:fbid', () => {
+    it('should return error if no participant found with fbid=fbid', async () => {
```
Comment (alykhank) [inline test/commitments-specs.js]: Change this to `should return 204`

File: test/commitments-specs.js [hunk 3]
```diff
@@ -8,17 +8,23 @@ beforeEach(function syncDB () {
...
       await koaRequest
-        .get('/commitments/participant/1')
+        .get('/commitments/participant/p0')
+        .expect(400)
```
Comment (alykhank) [inline test/commitments-specs.js]: `204` to match the `GET /participants/:fbid` test in `test/participants-specs.js`

**Pull request merged**